pages/common/xml-edit.md

```diff
@@ -0,0 +1,28 @@
+# xml-edit
+
+> Edit an XML document.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- TBD:
+
+`xml edit`
+
+- TBD:
+
+`xml edit`
+
+- TBD:
+
+`xml edit`
+
+- TBD:
+
+`xml edit`
+
+- TBD:
+
+`xml edit`
```

Comment:
These seem like placeholders? Perhaps it might be worth either:

1. Filling these in
2. Removing this page from the PR and opening a separate PR at a later time

Reply:
Please see the PR comment. I will complete them soon, workload permitting. I ran out of weekend.